### DISCUSSION TITLE
🎵 God Save The Queen 🎵

### DISCUSSION THREAD

**Discussion by j4james** [microsoft/terminal#13220] [Show and tell]
In honour of the Queen's Platinum Jubilee, and the "Play Sound" PR recently being merged, I made a cheesy tune for the console.

https://user-images.githubusercontent.com/4181424/171716919-c2665863-8667-4d7f-a82e-4ffc030582b4.mp4

**Comment by mobluse** [comment]
Could you give the printf "\e[...,~" command for "God Save the Queen"?

**Reply by j4james** [reply]
Here you go...
```
printf "\e[3;18;6;6;8,~\e[3;27;5,~\e[3;9;6,~\e[3;18;8;10;10;11,~\e[3;27;10,~\e[3;9;8,~\e[3;18;6;8;6;5,~\e[3;54;6,~\e[3;18;13;13;13,~\e[3;27;13,~\e[3;9;11,~\e[3;18;10;11;11;11,~\e[3;27;11,~\e[3;9;10,~\e[3;18;8;10,~\e[3;9;11;10;8;6,~\e[3;27;10,~\e[3;9;11,~\e[3;18;13,~\e[3;9;15;11,~\e[3;18;10;8,~\e[3;54;6,~"
```
